api/src/org/labkey/api/data/DbScope.java

```diff
@@ -1252,6 +1252,12 @@ public <OptionType extends SchemaTableOptions> SchemaTableInfo getTable(OptionTy
         return getTableInfoCache(options.getSchema().getType()).get(options);
     }
 
+    // This is expensive... use only for debugging and troubleshooting

```

Comment:
Using JavaDoc makes it more likely people will see this comment in statement completion, etc

```suggestion
    /** This is expensive... use only for debugging and troubleshooting */
```